streams.yml

```diff
@@ -81,94 +85,41 @@ ibm-rhel-8-golang-1.21:
   mirror_manifest_list: true
   upstream_image: quay.io/openshift-release-dev/golang-builder--ibm-share:rhel-8-golang-1.21-openshift-{MAJOR}.{MINOR}
 
-golang-1.20:
-  aliases:
-  - rhel-8-golang-{GO_PREVIOUS}
-  image: openshift/golang-builder:v1.20.12-202403212140.el8.g2983c24.el8
-  mirror: true
-  transform: rhel-8/golang
-  # Leave this upstream information in place even if mirror/transform is disabled. Test Platform read the data.
-  upstream_image_base: registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-{MAJOR}.{MINOR}.art
-  upstream_image: registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-{MAJOR}.{MINOR}
-  # To ensure consistency in our builds and allow CI to use the same build, mirror the upstream builder image from ocp/builder to ocp-private/builder-priv.
-  upstream_image_mirror:
-  - registry.ci.openshift.org/ocp-private/builder-priv:rhel-8-golang-1.20-openshift-{MAJOR}.{MINOR}
-
-ibm-rhel-8-golang-1.20:
-  # Mirror non-embargoed golang builders for IBM
-  image: openshift/golang-builder:v1.20.12-202403212140.el8.g2983c24.el8
-  mirror: true
-  mirror_manifest_list: true
-  upstream_image: quay.io/openshift-release-dev/golang-builder--ibm-share:rhel-8-golang-1.20-openshift-{MAJOR}.{MINOR}
-
-rhel-9-golang-1.20:
-  aliases:
-  - rhel-9-golang-{GO_PREVIOUS}
-  image: openshift/golang-builder:v1.20.12-202403212137.el9.g144a3f8.el9
-  mirror: true
-  transform: rhel-9/golang
-  # Leave this upstream information in place even if mirror/transform is disabled. Test Platform read the data.
-  upstream_image_base: registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-{MAJOR}.{MINOR}.art
-  upstream_image: registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-{MAJOR}.{MINOR}
-  # To ensure consistency in our builds and allow CI to use the same build, mirror the upstream builder image from ocp/builder to ocp-private/builder-priv.
-  upstream_image_mirror:
-  - registry.ci.openshift.org/ocp-private/builder-priv:rhel-9-golang-1.20-openshift-{MAJOR}.{MINOR}
-
-ibm-rhel-9-golang-1.20:
-  # Mirror non-embargoed golang builders for IBM
-  image: openshift/golang-builder:v1.20.12-202403212137.el9.g144a3f8.el9
-  mirror: true
-  mirror_manifest_list: true
-  upstream_image: quay.io/openshift-release-dev/golang-builder--ibm-share:rhel-9-golang-1.20-openshift-{MAJOR}.{MINOR}
-
 # This image is not used by ART. It is an artifact required in upstream CI to build unit tests.
 # Our transform is designed to create a buildconfig atop a specific golang version, layering on
 # packages that upstream has traditionally had present in its build_roots.
-rhel-8-golang-1.20-ci-build-root:
+rhel-8-golang-1.21-ci-build-root:
```

Comment:
let's also include ibm and ci-build-root streams for 1.22 el8 and el9, so add 4 streams in total.